deploy_target_perception:
  ros__parameters:
    pcd_file: "/home/std/workspace/test_pkg_ws/src/docking_pose_estimator/template/depoly.pcd"    #部署标准点云模版
    pointcloud_topic: "/camera/points"
    deployed_template_cloud_topic: "/deployed_template_cloud"
    filtered_source_cloud_topic: "/filtered_source_cloud"
    save_command_topic: "/save_pointcloud_command"
    save_path: "/home/std/workspace/test_pkg_ws/src/docking_pose_estimator/template/deploy/"
    voxel_size: 0.05
    camera_frame: "camera_rgb_frame"
    robot_frame: "base_footprint"
    # 实时点云滤波参数
    x_min: 0.0
    x_max: 3.0
    y_min: -1.0
    y_max: 1.0
    z_min: 0.1
    z_max: 2.0
    # 模板点云滤波参数
    template_x_min: 0.0
    template_x_max: 3.0
    template_y_min: -1.0
    template_y_max: 1.0
    template_z_min: 0.0
    template_z_max: 2.0
    # 初始位姿
    initial_x: 0.0
    initial_y: 0.0
    initial_z: 0.0
    initial_roll: 0.0
    initial_pitch: 0.0
    initial_yaw: 0.0
    use_ndt: false